phoenix-pherf/src/main/java/org/apache/phoenix/pherf/workload/MultiThreadedDifferTest.java

```diff
@@ -0,0 +1,46 @@
+package org.apache.phoenix.pherf.workload;
```

Comment:
Tests should be in the `src/test/...<package>` directory. ITs in `src/IT/...<package>` and only actual code in the `src/main/...<package>`. You should move this to `phoenix-pherf/src/test/...`